src/scss/components/local-footer/_local-footer.scss

```diff
@@ -0,0 +1,37 @@
+@charset "UTF-8";
+
+.su-local-footer  {
+  .su-local-footer__social-google {
+    i {
+      &::before {
+        content: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' height='20' width='20' viewBox='0 0 512 512'%3E%3C!--!Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free Copyright 2023 Fonticons, Inc.--%3E%3Cpath d='M390.9 298.5c0 0 0 .1 .1 .1c9.2 19.4 14.4 41.1 14.4 64C405.3 445.1 338.5 512 256 512s-149.3-66.9-149.3-149.3c0-22.9 5.2-44.6 14.4-64h0c1.7-3.6 3.6-7.2 5.6-10.7c4.4-7.6 9.4-14.7 15-21.3c27.4-32.6 68.5-53.3 114.4-53.3c33.6 0 64.6 11.1 89.6 29.9c9.1 6.9 17.4 14.7 24.8 23.5c5.6 6.6 10.6 13.8 15 21.3c2 3.4 3.8 7 5.5 10.5zm26.4-18.8c-30.1-58.4-91-98.4-161.3-98.4s-131.2 40-161.3 98.4L0 202.7 256 0 512 202.7l-94.7 77.1z'/%3E%3C/svg%3E");
```

Comment:
You don't need all of this. it should be just a few characters. Google is `content: "\f1a0"`. You may need to apply the font family and font weight to get them to properly display though. You can inspect the icon on the [FA site](https://fontawesome.com/v5/search?q=google&o=r) and see the `content` property. Remember to use the FA 5 icon library, not 6+

Reply:
I did it this way because the [google scholar icon](https://fontawesome.com/icons/google-scholar?f=brands&s=solid) is only available on FA 6+, and not on v5 of FA.